cime_config/buildnml

```diff
@@ -322,12 +321,15 @@ def buildnml(case, caseroot, compname):
     create_namelist_infile(case, user_nl_file, infile)
     control_infile = [infile]
 
+    # NOTE: User definition *replaces* existing definition.
+    files = Files(comp_interface="nuopc")
+
     # Create the namelist generator object - independent of instance
-    definition_files = [srcroot + "/cime_config/namelist_definition_mizuRoute.xml"]
-    nmlgen = NamelistGenerator(case, definition_files)
+    definition_files = [os.path.join( srcroot,  "cime_config/namelist_definition_mizuRoute.xml") ]
```

Comment:
Just curious----
to form file path,  using `os.path join` is better than a simple string concatenation?

Reply:
make sense that windows machine is not work if use string concatenation including `/` (windows use `\` instead of `/`)